middleware/lang-resolver.js

```diff
@@ -0,0 +1,56 @@
+const i18n = require('electron-i18n')
+const url = require('url')
+var parseUrl = require('parseurl')
```

Comment:
You should be able to use Node's built-in `require('url').parse` here instead of adding a new dependency.

Reply:
I will try. Thank you for the feedback